### PR TITLE
[Python API] Add InputInfo and PreProcessInfo

### PR DESCRIPTION
**Pull request by akuporos**
25481
Deprecate inputs property of IENetwork and ExecutableNetwork and provide inputs_info property instead. This allowed to align Python API with C++ API and get access to PreProcessInfo.
Reviewed  id: !5772

### CODE REVIEW

File: inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api_impl_defs.pxd [hunk 1]
```diff
@@ -80,6 +116,25 @@ cdef extern from "<inference_engine.hpp>" namespace "InferenceEngine":
         const char *description
         apiVersion apiVersion
 
+    cpdef enum MeanVariant:
```
Comment (nkogteva) [inline inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api_impl_defs.pxd:119]: It seems that C.MeanVariant, C.ResizeAlgorithm and C.ColorFormat are not used anymore and can be safely removed from here.

Comment (akuporos) [reply inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api_impl_defs.pxd:119]: They used as returned types in the PreProcessInfo. But their values can be removed.

Comment (akuporos) [reply inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api_impl_defs.pxd:119]: @nkogteva fixed

**Comment by azhogov** [comment]
ci/jenkins passed, but failed to update staus

**Pull request merged**